src/components/DataTable/TableToolbar.js

```diff
@@ -4,16 +4,14 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
+import React from 'react';
 import { settings } from 'carbon-components';
-import wrapComponent from '../../tools/wrapComponent';
 
 const { prefix } = settings;
 
-const TableToolbar = wrapComponent({
-  name: 'TableToolbar',
-  type: 'section',
-  className: `${prefix}--table-toolbar`,
-});
-
+const TableToolbar = ({ children }) => (
+  <section role="group" className={`${prefix}--table-toolbar`}>
```

Comment:
@vpicone I can open those tickets no problem! 

We're close! `<section>` just needs two more attributes to make it sing👩🏽‍🎤: 
```suggestion
  <section role="group" aria-label="data table toolbar" tabIndex="0" className={`${prefix}--table-toolbar`}>
```

With the above -- our grouping of tools is keyboard navigable and will sound like _"main, data table toolbar, group. you are currently on a group --"_ with VoiceOver and variations on that theme for NVDA, JAWS

Reply:
I'm not sure if this is a bugged I missed yesterday morning, or if this is newly introduced via recent changes, but the overflow menu button has _even stranger_ behavior:

![Apr-23-2019 00-55-56](https://user-images.githubusercontent.com/40970507/56558360-0ebe5380-6564-11e9-9005-e7244abf0581.gif)

When the menu button has focus the user is instructed to move through the menu with the left and right arrow keys -- at which point they find the only other keyboard navigable item is the cog svg 😕

Reply:
Yeah it's just a wrapper for the OverflowMenu with a different default icon. There's a _lot_ of open issues around that component I think it should take priority for the next time someone has some extra time.